### DISCUSSION TITLE
Higher cpu and memory after latest beta upgrade (0.14.0 Beta 1)

### DISCUSSION THREAD

**Discussion by krustophenia** [blakeblackshear/frigate#11557] [General]
Is anyone else experiencing this? Looks quite unstable for now (and yeah, I know what beta means). Just a heads up.

(screenshots before and after upgrade)

<img width="676" alt="Screenshot 2024-05-27 at 14 37 46" src="https://github.com/blakeblackshear/frigate/assets/39946234/a87cf80f-35f4-46d5-ae18-5167536e4c96">
<img width="681" alt="Screenshot 2024-05-27 at 14 37 36" src="https://github.com/blakeblackshear/frigate/assets/39946234/5601d3cb-a5de-45b8-83bc-1f47400d04de">

**Comment by NickM-27** [comment]
it would be best if you provided screenshots of the system metrics pages. In general I have not seen an increase. Seeing your config would also be helpful

**Comment by Mincka** [comment]
Test if you see a different workload with and without the web interface open.
See https://github.com/blakeblackshear/frigate/discussions/11528
